site/help-text/run-help.txt

```diff
@@ -18,8 +18,10 @@ OPTIONS:
 
 RUNNER OPTIONS:
         --no-run                    Compile, but don't run tests
-    -j, --test-threads <THREADS>    Number of tests to run simultaneously [possible values: integer
-                                    or "num-cpus"] [env: NEXTEST_TEST_THREADS=] [aliases: jobs]
+    -j, --test-threads <THREADS>    Number of tests to run simultaneously [possible values: integer or "num-cpus". 
```

Comment:
this change is not modified in dispatch.rs, so ./scripts/regenerate-help-text.sh reverted the change, I going to fix in #518 

@sunshowers